src/main/kotlin/it/pagopa/wallet/scheduler/service/RedisResumePolicyService.kt

```diff
@@ -0,0 +1,42 @@
+package it.pagopa.wallet.scheduler.service
+
+import it.pagopa.wallet.scheduler.config.properties.RedisResumePolicyConfig
+import java.time.Duration
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@Service
+class RedisResumePolicyService(
+    @Autowired private val redisTemplate: TimestampRedisTemplate,
+    @Autowired private val redisResumePolicyConfig: RedisResumePolicyConfig
+) : ResumePolicyService {
+    private val logger = LoggerFactory.getLogger(RedisResumePolicyService::class.java)
+
+    override fun getResumeTimestamp(): Instant {
```

Comment:
Consider adding a String as method's input to pass to findByKeyspaceAndTarget method as second parameter

Reply:
done it here 13e819008aa9a134136c7ae5ba42caba884e9c93